whois-api/src/test/java/net/ripe/db/whois/api/rdap/RdapRequestValidatorTest.java

```diff
@@ -0,0 +1,92 @@
+package net.ripe.db.whois.api.rdap;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Lists;
+import net.ripe.db.whois.api.rdap.domain.*;
+import net.ripe.db.whois.common.dao.RpslObjectDao;
+import net.ripe.db.whois.common.ip.Ipv4Resource;
+import net.ripe.db.whois.common.iptree.Ipv4Entry;
+import net.ripe.db.whois.common.iptree.Ipv4Tree;
+import net.ripe.db.whois.common.iptree.Ipv6Tree;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import org.joda.time.LocalDateTime;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Collections;
+import java.util.List;
+import org.junit.rules.ExpectedException;
+import org.junit.Rule;
+
+import javax.ws.rs.BadRequestException;
+
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class RdapRequestValidatorTest {
+
+    @Mock
+    private RdapExceptionMapper rdapExceptionMapper;
+
+    private RdapRequestValidator validator;
+
+    @Rule
+    public ExpectedException expectedEx = ExpectedException.none();
+
+    @Before
+    public void setup() {
+        this.validator = new RdapRequestValidator(rdapExceptionMapper);
```

Comment:
You can use @InjectMocks for the validator field, this will inject the rdapExceptionMapper automatically. No need for a setup() method.